src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -34,24 +36,55 @@
 public class Helios {
 
   public static HashMap<String, String> tokenCache = new HashMap<String, String>();
+  private static RequestConfig requestConfig = RequestConfig.custom().setConnectTimeout(3000)
+      .setSocketTimeout(3000).build();
 
 
-  private Helios() {
-  }
+  private Helios() {}
 
   public static String getAccessToken(User user) {
     return getAccessToken(user.getUserName(), user.getPassword());
   }
 
+  public static void deleteAllTokens(User user) {
+    String heliosGetTokenURL = HeliosConfig.getUrl(HeliosConfig.HeliosController.USERS);
+
+    CloseableHttpClient httpClient =
+        HttpClientBuilder.create().disableCookieManagement().disableConnectionState()
+            .disableAutomaticRetries().build();
+
+    HttpDelete httpDelete =
+        new HttpDelete(String.format("%s/%s/tokens", heliosGetTokenURL, user.getUserId()));
+    httpDelete.setConfig(requestConfig);
+    httpDelete.setHeader("THE-SCHWARTZ", Configuration.getCredentials().apiToken);
+
+    CloseableHttpResponse response = null;
+    try {
+      try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1141)